### ISSUE TITLE
reboot groups

### ISSUE THREAD

**Issue by mischief** [coreos/container-linux-update-operator#17]
this issue is for discussion on implementing reboot groups, similar to what locksmith has today.

currently my idea is to create a kubernetes TPR that describes which nodes are in which groups, and how many nodes of that groupt to reboot at once.

**Comment by bjudiantara** [comment]
is it possible to simulate reboot groups by running several set of agents and operator each on different namespace? e.g. group1 will have agents and operator with namespace `group1`,   group2 will have agents and operator with namespace `group2`

**Comment by sdemos** [comment]
It seems like that would be possible, as long as you restrict the nodes that the agent can run on to only nodes in your "group", but I've never tried to do anything like that before. However, currently cluo only reboots one node at a time anyway, so the main reason that reboot groups exist in locksmith isn't actually relevant here unless we provide the ability to configure the number of machines rebooting at once.

**Comment by bjudiantara** [comment]
yes, the agent will be restricted into particular node using node selector, thus effectively it is possible two nodes on different group will reboot at the same time
